### DISCUSSION TITLE
Error

### DISCUSSION THREAD

**Discussion by kintarinrl** [teamalgoritma/community#742] [Q&A] [answered]
![image](https://github.com/teamalgoritma/community/assets/63953994/f7117fad-d840-4c92-a3f5-93f0ea19ed65)

**Comment by arkanayudhistira** [comment, marked as answer]
Dear @kintarinrl,

Error tersebut disebabkan oleh belum terinstallnya library `chardet` pada virtual environment yang digunakan, silahkan untuk melakukan penginstallan library tersebut pada virtual environment dengan menjalankan 2 command berikut pada anaconda prompt:

```
conda activate r-tensorflow
pip install chardet
```

Silahkan dicoba, semoga membantu!
